app/templates/_brief_applications_stats.html

```diff
@@ -1,4 +1,22 @@
 {% import "toolkit/statistics.html" as statistics %}
 
-{{ statistics.big_stat(started_brief_responses_count, "Started applications" if started_brief_responses_count != 1 else "Started application", id="started-applications") }}
-{{ statistics.big_stat(completed_brief_responses_count, "Completed applications" if completed_brief_responses_count != 1 else "Completed application", id="completed-applications") }}
+{{ statistics.big_stat(
+     started_brief_responses_stats["started_brief_responses_total"],
+     "Started applications" if started_brief_responses_stats["started_brief_responses_total"] != 1 else "Started application",
```

Comment:
Might it be a bit cleaner to move this logic out of the template and into the view? You could pass it in as kwarg to render template?